contracts/README.md

```diff
@@ -52,60 +52,62 @@ the CI for every release tag. In case you need to build them manually for some
+  cosmwasm/optimizer:0.15.0 ./contracts/crypto-verify
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_floaty",target=/code/contracts/floaty/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.13 ./contracts/floaty
+  cosmwasm/optimizer:0.15.0 ./contracts/floaty
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_hackatom",target=/code/contracts/hackatom/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.13 ./contracts/hackatom
+  cosmwasm/optimizer:0.15.0 ./contracts/hackatom
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_ibc_reflect",target=/code/contracts/ibc-reflect/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.13 ./contracts/ibc-reflect
+  cosmwasm/optimizer:0.15.0 ./contracts/ibc-reflect
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_ibc_reflect_send",target=/code/contracts/ibc-reflect-send/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.13 ./contracts/ibc-reflect-send
+  cosmwasm/optimizer:0.15.0 ./contracts/ibc-reflect-send
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_queue",target=/code/contracts/queue/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.13 ./contracts/queue
+  cosmwasm/optimizer:0.15.0 ./contracts/queue
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_reflect",target=/code/contracts/reflect/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.13 ./contracts/reflect
+  cosmwasm/optimizer:0.15.0 ./contracts/reflect
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_staking",target=/code/contracts/staking/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.13 ./contracts/staking
+  cosmwasm/optimizer:0.15.0 ./contracts/staking
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_virus",target=/code/contracts/virus/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.13 ./contracts/virus
+  cosmwasm/optimizer:0.15.0 ./contracts/virus
 
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="devcontract_cache_empty",target=/code/contracts/empty/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.13 ./contracts/empty
+  cosmwasm/optimizer:0.15.0 ./contracts/empty
```

Comment:
I guess those target mounts are wrong now because the optimizer sets `--target-dir=/target`. We should check/fix this before merging.